101-confidential-ledger-create/azuredeploy.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "ledgerName": {
+            "type": "string",
+            "metadata": {
+                "description": "Ledger Name"
+            },
+            "minLength": 3,
+            "maxLength": 24
+        },
+        "principalId": {
+            "type": "string",
+            "metadata": {
+                "description": "Oid of the user"
+            },
+            "minLength": 18,
```

Comment:
min/max should be the same since a GUID is required

Reply:
Also move the sample to the quickstarts folder and remove 101 from the folder name

Reply:
Removed min/max entirely, moved to quickstart folder.